aat/src/resources/features/F-018/S-001.td.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "should invoke activity service And return response on successful requests",
+  "_guid_": "S-001",
+  "_extends_": "F-018_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is to add an activity for the case created above"
+  ],
+
+
+    "expectedResponse": {
+      "_extends_": "Common_201_Response",
+      "responseMessage": "OK",
```

Comment:
This should ideally be "Created". If it's returning OK for now - so be it. But please open an improvement ticket.

Reply:
It is returning Created. Deleted "responseMessage" from S-001.td.json so that it inherits the response message from Common_201_Response object.